### ISSUE TITLE
Load auto post URL on scroll

### ISSUE THREAD

**Issue by Eldenroot** [WildcardSearch/ScrollOn#19]
Would be nice to integrate something like this:
https://community.mybb.com/mods.php?action=view&pid=683

It adds automatically post url into the web browser when you see the post. Quite usefull :)

**Comment by WildcardSearch** [comment]
If it is okay with you, we can come back to this after we establish basic functionality.

**Comment by Eldenroot** [comment]
Yes, of course! First of all you need to release fully working and "usable" v1.0, then you can add more features. I just had this idea in my mind :)
